### DISCUSSION TITLE
GUI for pytest.

### DISCUSSION THREAD

**Discussion by LaserPhaser** [pytest-dev/pytest#8068] [Ideas]
Hi all 
Do we (in community) need GUI runner for pytest. I have my local project that works fine for my local tasks - mostly web API testing. 
If the community is interested in it I could make it opensource in 1 or 2 weeks and transfer to the pytest-dev.

**Comment by The-Compiler** [comment]
FWIW there are various things in that direction already:

- Various IDE integrations (e.g. PyCharm)
- [pytest_commander](https://github.com/ryanc414/pytest_commander) (Web UI)
- [pytest-gui](https://github.com/tfio/pytest-gui) (GUI? Seems like abandoned?)
- [pytest-ui](https://github.com/martinsmid/pytest-ui) (TUI)

Other than the IDE integrations, I don't think any of those really gained any popularity though.
